initial.c

```diff
@@ -0,0 +1,13 @@
+#include <initials.c> /*Sheqi Zhang 2016/06/29*/
```

Comment:
You shouldn't include the same file that your code is in. It will return an error. Instead, you should have included <stdio.h> since you're printing in this assignment.